Sources/CardPayments/CardClientError.swift

```diff
@@ -28,6 +28,15 @@ enum CardClientError {
 
         /// 6. GraphQLClient is unexpectedly nil
         case nilGraphQLClientError
+        
+        /// 7. An error from 3DS verification
+        case threeDSVerificationError
+        
+        /// 8. Missing Deeplink URL from 3DS
+        case missingDeeplinkURLError
```

Comment:
Take it or leave it comment - we usually call the URL that we get _back_ from a service the `returnURL`